### PR TITLE
Instructions to build in Docker

### PR DESCRIPTION
**Pull request by jglick**
I am not a Go developer and had never heard of Glide until half an hour ago, so take this suggested command with several grains of salt. ~(In particular, I am unsure if `glide.lock` is supposed to be versioned or not.)~ That said, it does seem to work.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
-vendor
\ No newline at end of file
+vendor
+.glide
```
Comment (ashleyschuett) [inline .gitignore]: please remove from `.gitignore`

File: .gitignore [hunk 2]
```diff
@@ -1 +1,4 @@
...
+glide.lock
```
Comment (ashleyschuett) [inline .gitignore]: please remove from `.gitignore`

**Comment by jglick** [comment]
Regarding `.glide` and `glide.lock`: these are generated by the build (for me!), but not present in the repository, so I was unsure if these were: files which were meant to be versioned but which you just forgot to `git add`; or files which are generated by each local build and need not be versioned (my original guess); or files which are not supposed to be there at all (in which case my build instructions may be wrong).

**Comment by ashleyschuett** [comment]
thank you!

**Pull request merged**